fec.config.js

```diff
@@ -1,11 +1,16 @@
 const path = require('path');
 
 module.exports = {
-  appUrl: ['/settings/learning-resources', '/openshift/learning-resources'],
+  appUrl: ['/settings/learning-resources', '/openshift/learning-resources', "/staging/global-learning-resources-page"],
```

Comment:
I don't think it should be /staging/. Check with kathryn Dixon about the url

Reply:
We do not know the URL yet. We will add it to staging for development/tesring purposes.